packages/next/bin/next.ts

```diff
@@ -105,6 +105,19 @@ if (process.env.NODE_ENV) {
 
 ;(process.env as any).NODE_ENV = process.env.NODE_ENV || defaultEnv
 
+// x-ref: https://github.com/vercel/next.js/pull/34688#issuecomment-1047994505
+if (process.versions.pnp === '3') {
+  const nodeVersionParts = process.versions.node
+    .split('.')
+    .map((v) => Number(v))
+
+  if (nodeVersionParts[0] < 16 || nodeVersionParts[1] < 14) {
```

Comment:
```suggestion
  if (nodeVersionParts[0] < 16 || (nodeVersionParts[0] === 16 && nodeVersionParts[1] < 14)) {
```

This might have a false positive for Node.js 17